tools/src/test/java/org/apache/kafka/tools/GetOffsetShellTest.java

```diff
@@ -182,14 +187,19 @@ public void testTopicPartitionsArg() {
         setUp();
 
         List<Row> offsets = executeAndParse("--topic-partitions", "topic1:0,topic2:1,topic(3|4):2,__.*:3");
-        List<Row> expected = Arrays.asList(
-                new Row("__consumer_offsets", 3, 0L),
+        ArrayList<Row> expected = new ArrayList<>(
```

Comment:
It seems we can keep using `List<Row>`, right?

Reply:
Sure - I've changed to use `List<Row>`